### ISSUE TITLE
Unapproved names show as approved in some areas

### ISSUE THREAD

**Issue by miningslayer18** [Solero/Houdini#18]
If I have a penguin who's name is not approved, if I open my igloo, my name shows as approved on the open igloos list. Same thing happens when playing sled race or on friends lists.

**Comment by Spydar007** [comment]
Yes, this is a bug and will be fixed in the new Houdini-asyncio version.
